packages/zosfiles/__tests__/__system__/cli/mount/fs/cli.files.mount.fs.system.test.ts

```diff
@@ -0,0 +1,169 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Session } from "@zowe/imperative";
+import { getUniqueDatasetName, runCliScript } from "../../../../../../../__tests__/__src__/TestUtils";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { ITestPropertiesSchema } from "../../../../../../../__tests__/__src__/properties/ITestPropertiesSchema";
+import { Delete } from "../../../../../src/api/methods/delete";
+
+const ZOWE_OPT_BASE_PATH = "ZOWE_OPT_BASE_PATH";
+
+let REAL_SESSION: Session;
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+let TEST_ENVIRONMENT_NO_PROF: ITestEnvironment;
+let defaultSystem: ITestPropertiesSchema;
+let volume: string;
+
+describe("Mount and unmount file system", () => {
+    let fsname: string;
+    let mountPoint: string;
+
+    // Create the unique test environment
+    beforeAll(async () => {
+        TEST_ENVIRONMENT = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "zos_mount_unmount_file_system"
+        });
+
+        defaultSystem = TEST_ENVIRONMENT.systemTestProperties;
+        volume = defaultSystem.datasets.vol;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(TEST_ENVIRONMENT);
+        fsname = getUniqueDatasetName(defaultSystem.zosmf.user);
+
+        const dirname = getUniqueDatasetName(defaultSystem.zosmf.user).split(".")[1];
+        mountPoint = "//tmp/" + dirname;
+        const sshCommand = "mkdir " + mountPoint;
+
+        const response = runCliScript(__dirname + "/__scripts__/command/command_setup.sh",
```

Comment:
It's totally fine how you have it, but in the `before` or `after` blocks, you can use the APIs instead of running a command.